napari/layers/_layer_actions.py

```diff
@@ -171,6 +189,14 @@ def _projdict(key) -> ContextAction:
         'show_when': 'True',
     }
 
+def _labeltypedict(key) -> ContextAction:
+    return {
+        'description': key,
+        'action': partial(_convert_dtype, mode=key),
+        'enable_when': 'not {}_label'.format(key),
```

Comment:
combined with `active_layer_dtype` we can also limit it to labels here as well.

```suggestion
        'enable_when': f'only_labels_selected and active_layer_dtype != {key!r}',
```